src/styles/annotator/notebook.scss

```diff
@@ -24,4 +24,23 @@
   padding: 0;
   width: 100%;
   height: 100%;
+  border: none;
+}
+
+.Notebook__controller-bar {
```

Comment:
A comment or two here about what the intended purpuse/layout is for this element would be helpful for the future.

Reply:
Added.